core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java

```diff
@@ -340,12 +340,7 @@ public boolean isValid(String value) {
         if (!isValidQuery(uri.getRawQuery())) {
             return false;
         }
-
-        if (!isValidFragment(uri.getRawFragment())) {
-            return false;
-        }
-
-        return true;
+        return isValidFragment(uri.getRawFragment());
```

Comment:
https://github.com/jenkinsci/jenkins/blob/c53b5bac8da3764973111f153e9eb1e6ea3b6112/core/src/main/java/jenkins/org/apache/commons/validator/routines/UrlValidator.java#L17 makes it clear that changes from upstream should be marked as such to make it easier to keep this in sync.

That said, this doesn't seem relevant enough to justify deviating from upstream.

Reply:
Thanks for reminding,i ignored it